src/components/SiteMenu.js

```diff
@@ -20,7 +20,7 @@ import { Nav, Search } from "@adobe/parliament-ui-components"
 
 import "./sitenav.css"
 
-const SiteMenu = ({ currentPage, pages }) => {
+const SiteMenu = ({ currentPage, pages , depth = 3 }) => {
```

Comment:
@macdonst this _seems_ to have been the original default. Not sure if it's still valid?